### DISCUSSION TITLE
Label the buttons

### DISCUSSION THREAD

**Discussion by deandonaldson** [Pirate-MIDI/Pirate-MIDI-Features-Bug-Tracking#166] [Editor App Feature Requests]
Super simple, but can you denote which button is which... just a simple 1, 2, 3, etc in the middle of the buttons please. 

helps program when the Bridge is not in sight ;-)

cheers!
D

**Comment by simonaglover95** [comment]
Hi @deandonaldson as soon as you click on a switch, the menu that opens tells you which switch you're on (top left - pic attached). If you accidentally clicked on the wrong switch you can quickly move to the correct switch by pressing the "next switch" or "previous switch" buttons at the bottom of the switches menu that opens. Also, the default bank labels actually label the switches, so if you can't remember you can flick to a bank that is in a default state. Hopefully after using the device for a while you'll commit the switch positions to memory and this won't be much of an issue.
<img width="814" alt="image" src="https://github.com/Pirate-MIDI/Pirate-MIDI-BridgeOS/assets/59382082/1445306f-e591-4370-8b9c-add11697a766">
